### DISCUSSION TITLE
Include discussions/activities?

### DISCUSSION THREAD

**Discussion by jmgirard** [PittMethods/appliedml#19] [General]
Do we still want to include activities and discussions this year? If so, do we want them built into each section or only at certain time points? I think a few well timed and designed activities could add some value, but would also be fine cutting them (as we did not mention activities in the advertisement). Also, people work at such different speeds that it's difficult to assign a specific amount of time for them. The way I tried to handle this in my R workshop this year was to only have activities right before lunch and right before ending for the day. Then people could take extra time if they wanted to without eating into the true workshop time (or skip them entirely).

**Comment by jmgirard** [comment]
For more conceptual lectures, I think a few comprehension check questions works really nicely. This is what I'm planning for 1A and 2B. But for more skills-focused lectures, I think live coding and/or practice activities might make more sense.

**Comment by ShirleyBWang** [comment]
Sounds good! I need to add in comprehension checks still. I was planning to have live coding for 1D and a hands-on coding activity for 2D so far. I don't think we need them *every* time like we did last year, but having a few throughout makes sense to me!
